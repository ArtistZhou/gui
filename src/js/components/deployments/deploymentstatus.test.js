import React from 'react';
import renderer from 'react-test-renderer';
import DeploymentStatus from './deploymentstatus';

describe('DeploymentStatus Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DeploymentStatus refreshStatus={jest.fn()} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
