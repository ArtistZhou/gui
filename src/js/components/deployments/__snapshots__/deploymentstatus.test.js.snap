// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeploymentStatus Component renders correctly 1`] = `
<div>
  <div
    className="results-status"
  >
    <div
      aria-label="Skipped"
      className="hint--bottom disabled"
    >
      <span
        className="status skipped"
      >
        0
      </span>
    </div>
    <div
      aria-label="Pending"
      className="hint--bottom disabled"
    >
      <span
        className="status pending"
      >
        0
      </span>
    </div>
    <div
      aria-label="In progress"
      className="hint--bottom disabled"
    >
      <span
        className="status inprogress"
      >
        0
      </span>
    </div>
    <div
      aria-label="Successful"
      className="hint--bottom disabled"
    >
      <span
        className="status success"
      >
        0
      </span>
    </div>
    <div
      aria-label="Failed"
      className="hint--bottom disabled"
    >
      <span
        className="status failure"
      >
        0
      </span>
    </div>
  </div>
</div>
`;
